### PR TITLE
Improvements about assertEquals and namespaces

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log

- To let assertion equals checking clearly, it should use `assertSame` to make assert equals checking strict.
- Fix namespaces about `Migrations\Test\TestCase\*`.

### CODE REVIEW

**Comment by dereuromark** [comment]
There is a small conflict

**Comment by peter279k** [comment]
> There is a small conflict

Thanks for your mentioning. I will fix that.

**Comment by dereuromark** [comment]
ping @peter279k

**Comment by peter279k** [comment]
@dereuromark. The conflicts have been resolved.

File: tests/TestCase/View/Helper/MigrationHelperTest.php [hunk 1]
```diff
@@ -262,22 +262,22 @@ public function testColumn()
...
-        $this->assertEquals(1, $this->helper->value(1));
-        $this->assertEquals(-1, $this->helper->value(-1));
-        $this->assertEquals(1.5, $this->helper->value(1.5));
-        $this->assertEquals(1.5, $this->helper->value('1.5'));
-        $this->assertEquals(1, $this->helper->value('1'));
+        $this->assertSame('null', $this->helper->value(null));
+        $this->assertSame('null', $this->helper->value('null'));
+        $this->assertSame('true', $this->helper->value(true));
+        $this->assertSame('false', $this->helper->value(false));
+        $this->assertSame(1, $this->helper->value(1));
```
Comment (markstory) [inline tests/TestCase/View/Helper/MigrationHelperTest.php:269]: This test [is failing](https://travis-ci.com/github/cakephp/migrations/jobs/399765313#L435)

Comment (peter279k) [reply tests/TestCase/View/Helper/MigrationHelperTest.php:269]: @markstory, thanks for your review.

Let me fix them :).

Comment (peter279k) [reply tests/TestCase/View/Helper/MigrationHelperTest.php]: Failed assertion has been fixed now.

**Pull request merged**